Debugging.md

```diff
@@ -0,0 +1,70 @@
+# Debugging your Game Servers
+
+Sometimes your game servers cannot start and the Builds are showing up as Unhealthy. In this case, you'd like to see debug your game server running on PlayFab Multiplayer Servers Virtual Machine. Or, sometimes, even though your game server is running smoothly, you may need to connect to the Virtual Machine (VM) and see various statistics/logs about your game server.
+
+In order to connect to the VM hosting your game server (either Windows or Linux), you can get RDP/SSH credentials using [CreateRemoteUser](https://docs.microsoft.com/en-us/rest/api/playfab/multiplayer/multiplayerserver/createremoteuser?view=playfab-rest) API call or using the "Connect" button on playfab.com web application.
+
+As soon as you connect to the VM, you can use the console of the operating system to monitor your game servers. Remember that MPS service is natively using Docker containers on both Windows and Linux to spin up game server processes. To run Docker CLI commands, you'll need an admin powershell on Windows and probably need to `sudo su -` on Linux.
+
+**These are advanced Docker container debugging instructions, usage of them might break your game servers. You should not use these instructions unless you absolutely know what you're doing**.
```

Comment:
>unless you absolutely know what you're doing [](start = 145, length = 44)

maybe too strong. Maybe, "you should not use these on servers handling retail traffic?"